tile/tile_magma.py

```diff
@@ -35,14 +35,19 @@ def __init__(self, core):
             tile_id=magma.In(magma.Bits(16)),
             clk=magma.In(magma.Clock),
             reset=magma.In(magma.AsyncReset),
-            read_config_data=magma.Out(magma.Bits(32))
+            read_config_data=magma.Out(magma.Bits(32)),
+            stall=magma.In(magma.Bits(4))
         )
 
         self.wire(self.ports.north, self.sb.ports.north)
         self.wire(self.ports.west, self.sb.ports.west)
         self.wire(self.ports.south, self.sb.ports.south)
         self.wire(self.ports.east, self.sb.ports.east)
 
+        # this is hacky, but connect stall if the core has a stall input
+        if "stall" in self.core.ports:
```

Comment:
we could make this a function in `Core`: `has_stall()`. But its fine like this now.